### DISCUSSION TITLE
Simple Notes Box

### DISCUSSION THREAD

**Discussion by ashtom50** [advplyr/audiobookshelf#2109] [Ideas]
I mostly listen to Non Fiction audiobooks. The biggest issue with listening instead of reading a book is that I cant highlight keep sections. While this is a wider issue with audiobooks it would be great to get a simple notes text box that we can write in to note down key concepts. Maybe this could be expanded further in the future for better note taking with automatic added timestamps, sync with other apps ect.

**Comment by nichwall** [comment]
Two related feature requests (not sure if planned due to lots of features being requested in recent months)

https://github.com/advplyr/audiobookshelf/issues/1890
https://github.com/advplyr/audiobookshelf/issues/2013

There is also a bookmarks feature already implemented, but I think you need to be connected to the server (does not work offline)
